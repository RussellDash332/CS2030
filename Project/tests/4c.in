1 2 2 20
0 3.131408
0.313508 1.037533
1.204909 6.995223
2.776498 0.142237
3.876961 1.541726
3.909736 0.126590
9.00639 14.777172
9.04336 25.930199
9.105379 2.968470
9.159629 3.686675
9.224613 1.605316
10.147993 0.517321
11.204932 34.895950
12.428914 28.691500
13.109177 8.957896
15.263626 1.043020
15.524295 0.006333
15.939973 0.576351
17.793096 0.742370
18.765423 3.007573
0
3.138762
0.847804
0
0.848968
0
0
3.863139
0
0
25.460873
0
0
0
0
36.96365
0
3.577445
2.169913
0