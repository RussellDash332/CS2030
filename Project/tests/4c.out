0.000 1 arrives
0.000 1 serves by server 1
0.314 2 arrives
0.314 2 serves by self-check 2
1.205 3 arrives
1.205 3 serves by self-check 3
1.351 2 done serving by self-check 2
2.776 4 arrives
2.776 4 serves by self-check 2
2.919 4 done serving by self-check 2
3.131 1 done serving by server 1
3.877 5 arrives
3.877 5 serves by server 1
3.910 6 arrives
3.910 6 serves by self-check 2
4.036 6 done serving by self-check 2
5.419 5 done serving by server 1
8.200 3 done serving by self-check 3
9.006 7 arrives
9.006 7 serves by server 1
9.043 8 arrives
9.043 8 serves by self-check 2
9.105 9 arrives
9.105 9 serves by self-check 3
9.160 10 arrives
9.160 10 waits at server 1
9.225 11 arrives
9.225 11 waits at server 1
10.148 12 arrives
10.148 12 waits at self-check 2
11.205 13 arrives
11.205 13 waits at self-check 2
12.074 9 done serving by self-check 3
12.074 12 serves by self-check 3
12.429 14 arrives
12.429 14 waits at self-check 2
12.591 12 done serving by self-check 3
12.591 13 serves by self-check 3
13.109 15 arrives
13.109 15 waits at self-check 2
15.264 16 arrives
15.264 16 leaves
15.524 17 arrives
15.524 17 leaves
15.940 18 arrives
15.940 18 leaves
17.793 19 arrives
17.793 19 leaves
18.765 20 arrives
18.765 20 leaves
23.784 7 done serving by server 1
24.631 10 serves by server 1
28.318 10 done serving by server 1
28.318 11 serves by server 1
29.923 11 done serving by server 1
34.974 8 done serving by self-check 2
34.974 14 serves by self-check 2
47.487 13 done serving by self-check 3
47.487 15 serves by self-check 3
56.445 15 done serving by self-check 3
63.665 14 done serving by self-check 2
[6.320 15 5]
